app/models/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,11 @@
+module CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def remote_delete
+    raw_delete
+  end
+
+  def raw_delete
```

Comment:
Yeah :-1: on adding `remote_delete`, `Class#your_method` should run any common checks like policies then call `Class#raw_your_method`.
[`VmOrTemplate::Operations::Power`](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template/operations/power.rb) has some good examples as well as [orchestration_stack.rb#L78-L84](https://github.com/ManageIQ/manageiq/blob/master/app/models/orchestration_stack.rb#L78-L84) (which again show conflicting naming scheme :laughing:)

I don't see one scheme being used overwhelmingly, but personally I like the more descriptive method names.